platforms/web/types/global.d.ts

```diff
@@ -7,3 +7,9 @@ interface Window {
 interface HTMLDivElement {
   inert: boolean;
 }
+
+type OTTConfig = {
```

Comment:
This name to me implies something very global. Perhaps it should be something like `WebAppConfig`? It's also not used outside of `/web`.

Reply:
We could theoretically reuse it for more platforms, but then we want to define it somewhere else. Let's make it web-specific for now indeed...